package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "run-matlab-tests-action",
     "author": "The MathWorks, Inc.",
-    "version": "1.3.2",
+    "version": "1.3.3",
```

Comment:
I believe this version may be controlled by our automated build process, so no need to change this here.

Side-note: I would expect this to be a minor number bump rather than a patch number bump because it is adding features.